openmeetings-server/src/site/markdown/LdapAndADS.md

```diff
@@ -0,0 +1,238 @@
+    ldap_provisioning=AUTOCREATE
+    
+    # Ldap deref mode (never, searching, finding, always)
+    ldap_deref_mode=always
+    
+    #  Set this to 'true' if you want to use admin_dn to get user attributes
+    #  If any other value is set, user_dn will be used
+    ldap_use_admin_to_get_attrs=true
+    
+    # Ldap-password synchronization to OM DB
+    #  Set this to 'true' if you want OM to synchronize the user Ldap-password to OM's internal DB
+    #  If you want to disable the feature, set this to any other string.
+    #  Defautl value is 'true'
+    ldap_sync_password_to_om=true
+    
+    # Ldap group mode (NONE, ATTRIBUTE, QUERY)
+    # NONE means group associations will be ignored
+    # ATTRIBUTE means group associations will be taken from 'ldap_group_attr' attribute (M$ AD mode)
+    # QUERY means group associations will be taken as a result of 'ldap_group_query' query
+    ldap_group_mode=NONE
+    
+    ldap_group_query=(&(memberUid=%s)(objectClass=posixGroup))
+    
+    # Ldap user attributes mapping
+    # Set the following internal OM user attributes to their corresponding Ldap-attribute
+    ldap_user_attr_login=uid
+    ldap_user_attr_lastname=sn
+    ldap_user_attr_firstname=givenName
+    ldap_user_attr_mail=mail
+    ldap_user_attr_street=streetAddress
+    ldap_user_attr_additionalname=description
+    ldap_user_attr_fax=facsimileTelephoneNumber
+    ldap_user_attr_zip=postalCode
+    ldap_user_attr_country=co
+    ldap_user_attr_town=l
+    ldap_user_attr_phone=telephoneNumber
+    # optional attribute for user picture
+    #ldap_user_attr_picture=
+    ldap_group_attr=memberOf
+    
+    # optional, absolute URL will be used as user picture if #ldap_user_attr_picture will be empty
+    #ldap_user_picture_uri=picture_uri
+    
+    # Ldap ignore upper/lower case, convert all input to lower case
+    ldap_use_lower_case=false
+    
+    # Ldap import query, this query should retrieve all LDAP users
+    ldap_import_query=(objectClass=inetOrgPerson)
+    
+    ``` 
+    
+    NOTE: Directory Servers are individual for every organization, and
+    might use different attribute names than the ones listed in the
+    example config. Please obtain the information about the structure
+    and attribute names of the entries and adjust the config file
+    accordingly.
+
+    It is important to configure LDAP_SEARCH_PATH so that it points to the
+    node of the LDAP directory server that contains the users.
+
+    If there are multiple search paths you can either try to define a
+    search path that points to multiple nodes or define multiple LDAP
+    configurations in the LDAP administration pane of OpenMeetings.
+
+### 3. Configuring OpenMeetings to use the ldap config file
+
+The final step binds the config file of the previous section to
+OpenMeetings:
+
+-   Log into OpenMeetings as Administrator
+
+-   Open Administration->LDAP
+
+    <img src="../resources/images/ldap_screen.png"></img>
```

Comment:
URL should be `images/ldap_screen.png` (you can use markdown format as well)

Reply:
OK, how do I submit the change?

Reply:
please see above :)